openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -50,8 +51,37 @@ def process(self, instance):
                                ['override_viewport_options']
         )
         preset = lib.load_capture_preset(data=self.capture_preset)
-
+        # Grab capture presets from the project settings
+        capture_presets = self.capture_preset
+        # Set resolution variables from capture presets
+        width_preset = capture_presets["Resolution"]["width"]
+        height_preset = capture_presets["Resolution"]["height"]
+        # Set resolution variables from asset values
+        asset_width = instance.data.get("resolutionWidth")
+        asset_height = instance.data.get("resolutionHeight")
```

Comment:
I maybe mislead you. To get resolution from asset you should use 
```suggestion
        asset_data = instance.data["assetEntity"]["data"]
        asset_width = asset_data.get("width")
        asset_height = asset_data.get("height")
```
Content of `instance.data["resolutionWidth"]` is what is set in scene, which may not be 100% resolution from asset.

It could be maybe considered as last possible source for resolution?

Reply:
Thank you Kuba